### PR TITLE
Dialogs UI

### PR DESCRIPTION
**Pull request by falconedani**
**Staging:** https://dialogs-ui--labstorybook-master.netlify.app/?path=/docs/components-dialog--page
**Jira issue:** https://labcodes.atlassian.net/browse/DSYS-23
**Details**
Add styles with animations for both dialog types. Please take a closer look at animations and the method used to position the components. I could only achieve animations for when the open, but not when closing. Any tips are appreciated.

### CODE REVIEW

**Review by lucianoratamero — CHANGES_REQUESTED** [review]
both the styles and the code for this PR are kind of weird.
too many things to talk about here, though.
maybe shouwl we pair up on this, @falconedani ?

File: labsystem/scss/components/_lab-dialog.scss [hunk 1]
```diff
@@ -1,85 +1,353 @@
...
+  background-color: unset;
+  border: none;
+
+  &:focus {
+    border-radius: $radius-02;
+    box-shadow: 0 0 0 2px $teal-30;
+    background-color: transparent;
+    outline: unset;
+    
+    .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:158]: update to a more specific class

File: labsystem/scss/components/_lab-dialog.scss [hunk 2]
```diff
@@ -1,85 +1,353 @@
...
+      background-color: $teal-60;
+    }
+  }
+
+  &:hover {
+    border-radius: $radius-02;
+    background-color: $teal-10;
+
+    .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:167]: update to a more specific class

File: labsystem/scss/components/_lab-dialog.scss [hunk 3]
```diff
@@ -1,85 +1,353 @@
...
+      background-color: $teal-60;
+    }
+  }
+
+  .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:172]: update to a more specific class

File: labsystem/scss/components/_lab-dialog.scss [hunk 4]
```diff
@@ -1,85 +1,353 @@
...
+    background-color: $mineral-60;
+  }
+
+  &:active {
+    border-radius: $radius-02;
+    box-shadow: 0 0 0 2px $teal-30;
+    background-color: $teal-20;
+    outline: unset;
+    
+    .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:182]: update to a more specific class

File: labsystem/scss/components/_lab-dialog.scss [hunk 5]
```diff
@@ -1,85 +1,353 @@
...
+  display: flex;
+  justify-content: center;
+  margin-top: -$spacing-06;
+  margin-bottom: $spacing-01;
+  width: 100%;
+  height: 24px;
+  background-color: unset;
+  border: none;
+
+  .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:205]: update to a more specific class

File: labsystem/scss/components/_lab-dialog.scss [hunk 6]
```diff
@@ -1,85 +1,353 @@
...
+    background-color: $mineral-60;
+  }
+
+  &:focus {
+    border-radius: $radius-02;
+    box-shadow: 0 0 0 2px $teal-30;
+    background-color: transparent;
+    outline: unset;
+    
+    .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:215]: update to a more specific class

File: labsystem/scss/components/_lab-dialog.scss [hunk 7]
```diff
@@ -1,85 +1,353 @@
...
+      background-color: $teal-60;
+    }
+  }
+
+  &:hover {
+    border-radius: $radius-02;
+    background-color: $teal-10;
+
+    .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:224]: update to a more specific class

File: labsystem/scss/components/_lab-dialog.scss [hunk 8]
```diff
@@ -1,85 +1,353 @@
...
+      background-color: $teal-60;
+    }
+  }
+
+  .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:229]: update to a more specific class

File: labsystem/scss/components/_lab-dialog.scss [hunk 9]
```diff
@@ -1,85 +1,353 @@
...
+    background-color: $mineral-60;
+  }
+
+  &:active {
+    border-radius: $radius-02;
+    box-shadow: 0 0 0 2px $teal-30;
+    background-color: $teal-20;
+    outline: unset;
+    
+    .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:239]: update to a more specific class

File: labsystem/scss/components/_lab-dialog.scss [hunk 10]
```diff
@@ -1,85 +1,353 @@
...
+// Message Dialog elements
 .lab-dialog__icon {
   display: flex;
   justify-content: center;
+  align-items: center;
+  margin-top: -$spacing-07;
+  margin-bottom: $spacing-03;
+  height: 72px;
+  
+  .lab-icon {
```
Comment (falconedani) [inline labsystem/scss/components/_lab-dialog.scss:311]: update to a more specific class

File: labsystem/src/Dialog/StandardDialog.js [hunk 11]
```diff
@@ -103,11 +103,11 @@ export default class StandardDialog extends React.Component {
             onTouchStart={this.handleTouchStart}
             onTouchEnd={this.handleTouchEnd}
           >
-            <Icon type="collapse-open" size="petit" />
+            <Icon type="collapse-open" />
```
Comment (falconedani) [inline labsystem/src/Dialog/StandardDialog.js:106]: add classname

**Review by falconedani — COMMENTED** [review]
Changes needed on lab-icon class use

**Comment by lucianoratamero** [comment]
since the git tree is messed up, we'll create a new branch for these changes

**Pull request closed without merging**